ferrocene/doc/evaluation-plan/src/method.rst

```diff
@@ -25,14 +25,16 @@ the :doc:`safety-manual:index` with the following form:
 
    * - Error identifier 
      - An identifier for the error with format ERR_NAME_XXX where NAME is a short name describing the faulty feature, and XXX is a number.
-   * - Error description 
+   * - Description
      - The description of the error.
-   * - Condition of appearance 
+   * - Use case
```

Comment:
reproductibility?

Reply:
it can use better wording... I considered leaving it empty